ops/testing.py

```diff
@@ -1530,6 +1525,26 @@ def add_model_secret(self, owner: AppUnitOrName, content: Dict[str, str]) -> str
         model.Secret._validate_content(content)
         return self._backend._secret_add(content, owner_name)
 
+    def add_model_secret(self, owner: AppUnitOrName, content: Dict[str, str]) -> str:
+        """An alias for :meth:`add_charm_secret` for backward compatibility."""
+        return self.add_charm_secret(owner, content)
```

Comment:
For what it's worth, I think if we do change the name, then we should have a warning emitted here. Not sure whether `DeprecationWarning` or `PendingDeprecationWarning`.